idd/Energy+.idd.in

```diff
@@ -63475,6 +63547,7 @@ CoolingTower:VariableSpeed:Merkel,
   N6 , \field Design Water Flow Rate per Unit of Nominal Capacity
        \note This field is only used if the previous is set to autocalculate and performance input method is NominalCapacity
        \units m3/s-W
+       \ip-units gal/min-W
```

Comment:
@MatthewSteen I was thinking that the IP units should be gal-hr/min-Btu since the Nominal Capacity is expressed as Btu/hr in IP. If you agree, I will make the change while addressing #5443